include/IECorePython/RefCountedBinding.h

```diff
@@ -63,6 +63,27 @@ class RefCountedWrapper : public T, public WrapperGarbageCollector
 		template<typename Arg1, typename Arg2, typename Arg3>
 		RefCountedWrapper( PyObject *self, Arg1 arg1, Arg2 arg2, Arg3 arg3 );
 
+		template<typename Arg1, typename Arg2, typename Arg3, typename Arg4>
```

Comment:
To be clear, I was the c++11 change until after I purge all the color/image code, to avoid having to patch up dead code.